src/Microsoft.Diagnostics.DebugServices.Implementation/ThreadService.cs

```diff
@@ -59,6 +59,12 @@ public ThreadService(IServiceProvider services)
                     contextType = typeof(ArmContext);
                     break;
 
+                case (Architecture)9 /* Architecture.RiscV64 */:
```

Comment:
Can you add RiscV64 to Architecture?

Reply:
`Architecture Enum` (`using Architecture = System.Runtime.InteropServices.Architecture;`) is part of runtime. Already added into runtime sources (https://github.com/dotnet/runtime/blob/f28a8a72eee05e0b909a4f2cb58d66bc0f7a01fe/src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/Architecture.cs#L59), but will be allowed only  in next release.

Reply:
I wasn't thinking.  I forgot that we are using the runtime's Architecture enum.